BUILD

```diff
@@ -39,10 +39,9 @@ COPTS = select({
     ],
 })
 
-config_setting(
-    name = "msvc",
-    values = { "compiler": "msvc-cl" },
-)
+load(":compiler_config_setting.bzl", "create_compiler_config_setting")
```

Comment:
> I simply added only the hunterization commit via a rebase

I think you pull changes that we don't need. I'm expecting only CMake code touched, see previous hunterization:
* https://github.com/hunter-packages/protobuf/pull/7/files